tests/log-specs.js

```diff
@@ -3,7 +3,7 @@ import {log as logger, configure} from '..';
 describe('log', () => {
 	test('imperatively pass log extries into the analytics system', () => {
 		const log = jest.fn();
-		configure({enabled:true, log, idle: false});
+		configure({enabled:true, log, rules: [{}], idle: false});
```

Comment:
We can remove this change now that `undefined` rules results in base-case log-all, right?

Reply:
yes!

Reply:
Fixed!